files/main.css

```diff
@@ -383,6 +383,7 @@ iframe {
 	width: 100%;
 	height: 100%;
 	overflow: auto;
+	display: none;
```

Comment:
Heh... This made all the projects in https://threejs.org/ disappear 😅

Reply:
I've reverted this line for now: https://github.com/mrdoob/three.js/commit/4f4c845c5cc127f3c90b77417c0462d2e4813bc4

Would `display: unset;` work?

Reply:
@mrdoob I can't find the html file of the home page, it's not in the repo ?

We could replace this line with the following, since `#viewer` in the homepage is a `div` :
```css
iframe#viewer {
	display: none;
}
```

Reply:
> @mrdoob I can't find the html file of the home page, it's not in the repo ?

It's in the `gh-pages` branch.

> We could replace this line with the following, since `#viewer` in the homepage is a `div` :

Sounds good!